pkgs/top-level/all-packages.nix

```diff
@@ -15251,5 +15251,7 @@ let
   udevil = callPackage ../applications/misc/udevil {};
 
   facter = callPackage ../tools/system/facter {};
+  
+  fatresize = callPackage ../tools/filesystems/fatresize {};
```

Comment:
`all-packages.nix` is certainly a bit of a mess, but please move this to the `### TOOLS` section (respecting the more-or-less alphabetical order as much as possible).